meson_options.txt

```diff
@@ -3,9 +3,10 @@ option('logind-provider', type: 'combo', choices: ['auto', 'systemd', 'elogind']
 option('libseat', type: 'feature', value: 'auto', description: 'Enable support for rootless session via libseat')
 option('xcb-errors', type: 'feature', value: 'auto', description: 'Use xcb-errors util library')
 option('xwayland', type: 'feature', value: 'auto', yield: true, description: 'Enable support for X11 applications')
-option('x11-backend', type: 'feature', value: 'auto', description: 'Enable X11 backend')
 option('examples', type: 'boolean', value: true, description: 'Build example applications')
 option('icon_directory', description: 'Location used to look for cursors (default: ${datadir}/icons)', type: 'string', value: '')
 option('xdg-foreign', type: 'feature', value: 'auto', description: 'Enable xdg-foreign protocol')
 option('renderers', type: 'combo', choices: ['auto', 'gles2'], description:
        'Select the built-in renderer')
+option('backends', type: 'combo', choices: ['auto', 'drm', 'noop', 'headless', 'libinput', 'x11', 'wayland'],
```

Comment:
This should be an `array`, or else this only allows to select a single backend.

Reply:
yes, my fault